istioctl/cmd/internal-debug.go

```diff
@@ -37,7 +39,85 @@ const (
 	xdsPortName       = "https-dns"
 )
 
+type DebugCommandOpts struct {
+	// ConfigDistributed is used for receiving istio CRD resource, e.g. VirtualService/default/bookinfo
+	ConfigDistributed string
+}
+
+// AttachDebugCommandFlags attaches control-plane flags to the internal-debug Cobra command.
+// (Currently just --resource)
+func (o *DebugCommandOpts) AttachDebugCommandFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&o.ConfigDistributed, "resource", "",
+		"This Debug options is only used for config_distribution. And the --resource looks like this 'VirtualService/default/bookinfo'")
```

Comment:
"the resource to track using format kind/namespace/name, e.g.  `--resource VirtualService/default/bookinfo`  ")